geomstats/learning/radial_kernel_functions.py

```diff
@@ -0,0 +1,385 @@
+"""Radial kernel functions.
+
+References
+----------
+https://en.wikipedia.org/wiki/Kernel_(statistics)
+https://en.wikipedia.org/wiki/Radial_basis_function
+"""
+
+import geomstats.backend as gs
+
+
+def uniform_radial_kernel(distance, bandwidth=1.0):
```

Comment:
Could you add the mathematical formula and reference of each kernel? 

(It seems that some of them are not on the Wikipedia pages listed above?)